0009/README.md

```diff
@@ -0,0 +1,70 @@
+---
+id: 0009
+title: Automated GitHub Action Runner
+status: ideation 
+authors: David McKay <david@rawkode.com>
+---
+
+## Summary:
+
+The `tink` and `portal` repositories use GitHub Actions to run their automated tests on pull-request and push to master.
+
+A dedicated GitHub Action runner is provided for the [Tinkerbell organization](https://github.com/tinkerbell), this runner was manually created and provisioned by [Gianluca](https://github.com/gianarb). As this runner wasn't automated from the beginning, should anything happen to the server that requires it to be shutdown or migrated, it would prove timely and cumbersome to rebuild.
+
+This proposal is to use a best practice approach, Infrastructure as Code, to deliver the test infrastructure needed to support the Tinkerbell project. Doing so also provides the following benefits:
+
+- Complete transparency through open sourcing the automation to provision the runner
+- Anyone can contribute and help us maintain and improve the runner
+- Anyone can contribute runners using other cloud providers
+- Anyone can build and operate their own runner
+
+## Goals:
+
+- Automate the creation and provisioning of a dedicated GitHub Actions runner
+
```

Comment:
The following motivator was mentioned in the community meeting https://youtu.be/QzK_aJMYqpk (22m:40s @gianarb)
`- Host bots and other custom resources (such as triage-party)`